opt/jstachio-spring-example/src/main/java/module-info.java

```diff
@@ -27,30 +27,23 @@
  */
 @JStachePath(prefix = "views/", suffix = ".mustache") //
 module io.jstach.opt.spring.example {
-	requires transitive io.jstach.opt.spring.webmvc;
-	requires io.jstach.opt.jmustache;
-	requires com.samskivert.jmustache;
+	requires transitive io.jstach.opt.spring.boot.webmvc;
 
 	requires static spring.jcl;
-	requires static jakarta.servlet;
-	
-	requires spring.web;
```

Comment:
Because spring cannot be `requires transitive` in any of our modules you will need to leave those `requires spring.*` in.